nixos/modules/system/boot/loader/grub/install-grub.pl

```diff
@@ -562,11 +562,9 @@ sub addProfile {
             next;
         }
         my $date = strftime("%F", localtime(lstat($link)->mtime));
-        my $version =
-            -e "$link/nixos-version"
```

Comment:
This check can safely be removed because it's already being performed on line 560